CONTRIBUTING.md

```diff
@@ -1 +1,8 @@
-We require contributors to sign our Contributor License Agreement. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla. If you have any questions, please drop us a line at cla@fb.com. Thanks!
+
+# Contribution guidelines
+
+We ***require*** contributors to sign our Contributor License Agreement. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla. If you have any questions, please drop us a line at cla@fb.com. Thanks!
+
+## Submitting Pull-Requests
```

Comment:
Make it a first-level header? Also, capitalisation is inconsistent with the other header. `# Submitting pull requests`?